packages/xstate-scxml/test/fixtures/actionSend/send4.ts

```diff
@@ -9,14 +9,14 @@ const m = createMachine({
     },
```

Comment:
this PR should probably go over our SCXML-compatibility test suite and try to enable some new tests. Most of the tests are associated by the comments on why they are failing so locating the ones concerned with the changed behavior here should be easy to locate.

Reply:
I took a brief look and didn't immediately find any tests that this enabled.